Vagrantfile

```diff
@@ -1,65 +1,97 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-def host_bind_address
-  ENV['VAGRANT_INSECURE_FORWARDS'] =~ /^(y(es)?|true|on)$/i ? '*' : '127.0.0.1'
-end
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
```

Comment:
This seems unnecessary to me, I haven't seen any other Vagrantfile that has a variable for the API version. I'd just revert this to have the literal "2" below. :)